pgt/src/PGT/Output/TestSuite/examples/failure-on-empty-content.expected.parsed

```diff
@@ -0,0 +1 @@
+Failed reading: expected a valid test or database object definition but found none
```

Comment:
Comment: This example exercises the first condition in the newly added multi-way if in PGT.Output.TestSuite.parse